src/cswinrt/Directory.Build.props

```diff
@@ -0,0 +1,16 @@
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" >
+
+    <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+
+    <PropertyGroup>
+        <!--Avoid UniversalApiContract type not found error-->
+        <CppWinRTMergeNoValidate>true</CppWinRTMergeNoValidate>
```

Comment:
for context, was this needed because of the cppwinrt update or something else?

Reply:
originally, which were were going to use UniversalApiContract for SupportedOSPlatform support on interop.  in the end, we went with conditional compilation instead - will remove this